src/sentry/api/endpoints/relay/index.py

```diff
@@ -12,6 +13,9 @@
 
 @region_silo_endpoint
 class RelayIndexEndpoint(Endpoint):
+    publish_status = [
+        {"GET": ApiPublishStatus.UNKNOWN},
```

Comment:
why make this a list of dictionaries instead of just a dictionary?

Reply:
🤣 no reason. I don't know what happened in my brain.